src/Common/Core/Impl/Enums/OrientationEnum.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Common.Core {
+    public enum OrientationEnum {
```

Comment:
While this enum can be related to printing in general, it is still too specific to be referenced everywhere. For instance, editor assemblies don't need it, so I'd prefer to have it in `Microsoft.R.Components`